pkg/services/rpcsrv/server.go

```diff
@@ -1719,7 +1718,7 @@ func (s *Server) getStateRoot(ps params.Params) (any, *neorpc.Error) {
 
 func (s *Server) getStorage(ps params.Params) (any, *neorpc.Error) {
 	id, rErr := s.contractIDFromParam(ps.Value(0))
-	if rErr == neorpc.ErrUnknown {
+	if rErr == neorpc.ErrVerificationFailed {
```

Comment:
And I can't leave the comment in the proper place, but we have to return `ErrUnknownStorageItem` code from this method at L1735.

Reply:
It is in the separate commit

Reply:
In this commit then you have to change `neorpc.ErrUnknown` to `neorpc.ErrUnknownContract`, not to the `neorpc.ErrVerificationFailed`. Or another way is to move a separate commit that changes the behaviour to the first place in this PR, then you don't need to replace this error with some intermediate one.